addons.yaml

```diff
@@ -238,3 +238,10 @@ microk8s-addons:
       check_status: "daemonset.apps/fpga-operator"
       supported_architectures:
         - amd64
+
+    - name: "argocd"
+      description: "Argo CD is a declarative continuous deployment for Kubernetes."
+      version: "2.2.5"
+      check_status: "pod/argo-cd-server"
```

Comment:
```suggestion
      check_status: "deployment.apps/argo-cd-argcd-server"
```

I think the pod name is not correct here. This is the kubectl output for me:

```
# microk8s kubectl get pod,deploy -n argocd
NAME                                                         READY   STATUS    RESTARTS      AGE
pod/argo-cd-argocd-redis-58d4549574-fw68n                    1/1     Running   0             93s
pod/argo-cd-argocd-dex-server-5cb47f4755-hz2g5               1/1     Running   1 (83s ago)   93s
pod/argo-cd-argocd-server-5d8f4c89b7-ww9qh                   1/1     Running   0             93s
pod/argo-cd-argocd-application-controller-7b65595d88-kmmmf   1/1     Running   0             93s
pod/argo-cd-argocd-repo-server-7bd4cb988c-v2825              1/1     Running   0             93s

NAME                                                    READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/argo-cd-argocd-redis                    1/1     1            1           93s
deployment.apps/argo-cd-argocd-dex-server               1/1     1            1           93s
deployment.apps/argo-cd-argocd-server                   1/1     1            1           93s
deployment.apps/argo-cd-argocd-application-controller   1/1     1            1           93s
deployment.apps/argo-cd-argocd-repo-server              1/1     1            1           93s
```

With the suggested change, `microk8s status` shows ArgoCD as enabled:

```
# microk8s status --addon argocd
enabled
```

Reply:
Wow thanks for that one! good finding!